packages/client/src/app/store/store.ts

```diff
@@ -2,10 +2,23 @@ import { configureStore } from '@reduxjs/toolkit';
 import { userReducer } from './slices/userSlice';
 import { gameReducer } from './slices/gameSlice';
 
+export const reducer = { user: userReducer, game: gameReducer };
+
+declare global {
+    interface Window {
+        __PRELOADED_STATE__?: object;
+    }
+}
+
 export const store = configureStore({
-    reducer: { user: userReducer, game: gameReducer },
+    reducer,
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
если вам это правило нужен, то лучше добавьте так в конфиг:
```
eslint no-underscore-dangle: ["error", { "allow": ["__PRELOADED_STATE__"] }]
```
https://eslint.org/docs/latest/rules/no-underscore-dangle#allow

Reply:
Заодно и __SERVER_PORT__
```suggestion
eslint no-underscore-dangle: ["error", { "allow": ["__PRELOADED_STATE__", "__SERVER_PORT__"] }]
```